.travis.yml

```diff
@@ -14,6 +16,10 @@ matrix:
     - php: 7.0
   fast_finish: true
 
+cache:
+  directories:
+    - $HOME/.composer/cache
```

Comment:
@bocharsky-bw 

Travis suggests to use `$HOME` here, see http://docs.travis-ci.com/user/environment-variables/#Default-Environment-Variables:

>The following default environment variables are available to all builds.
>* `HOME=/home/travis` (do not depend on this value)